sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/api/Azure.Monitor.OpenTelemetry.LiveMetrics.net6.0.cs

```diff
@@ -0,0 +1,106 @@
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics
+{
+    public partial class LiveMetricsExporterOptions : Azure.Core.ClientOptions
+    {
+        public LiveMetricsExporterOptions() { }
+        public string ConnectionString { get { throw null; } set { } }
+        public Azure.Core.TokenCredential Credential { get { throw null; } set { } }
+        public bool EnableLiveMetrics { get { throw null; } set { } }
+    }
+    public static partial class LiveMetricsExtensions
```

Comment:
This should be an internal implementation and shouldn't be a new API. 
Please mark this class and methods as internal.